src/pages/workspace/expensifyCard/WorkspaceCardPageFeed.tsx

```diff
@@ -0,0 +1,74 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React from 'react';
+import {View} from 'react-native';
+import FeatureList from '@components/FeatureList';
+import type {FeatureListItem} from '@components/FeatureList';
+import * as Illustrations from '@components/Icon/Illustrations';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import type {FullScreenNavigatorParamList} from '@libs/Navigation/types';
+import AccessOrNotFoundWrapper from '@pages/workspace/AccessOrNotFoundWrapper';
+import WorkspacePageWithSections from '@pages/workspace/WorkspacePageWithSections';
+import CONST from '@src/CONST';
+import type SCREENS from '@src/SCREENS';
+
+const tripsFeatures: FeatureListItem[] = [
+    {
+        icon: Illustrations.MoneyReceipts,
+        translationKey: 'workspace.moreFeatures.expensifyCard.feed.features.cashBack',
+    },
+    {
+        icon: Illustrations.CreditCardsNew,
+        translationKey: 'workspace.moreFeatures.expensifyCard.feed.features.unlimited',
+    },
+    {
+        icon: Illustrations.MoneyWings,
+        translationKey: 'workspace.moreFeatures.expensifyCard.feed.features.spend',
+    },
+];
+type WorkspaceCardPageFeedProps = StackScreenProps<FullScreenNavigatorParamList, typeof SCREENS.WORKSPACE.EXPENSIFY_CARD>;
+
+function WorkspaceCardPageFeed({route}: WorkspaceCardPageFeedProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const theme = useTheme();
+    const {isSmallScreenWidth} = useWindowDimensions();
+
+    return (
+        <AccessOrNotFoundWrapper
+            policyID={route.params.policyID}
+            featureName={CONST.POLICY.MORE_FEATURES.ARE_EXPENSIFY_CARDS_ENABLED}
+        >
+            <WorkspacePageWithSections
+                shouldUseScrollView
+                icon={Illustrations.HandCard}
+                headerText={translate('workspace.common.expensifyCard')}
+                route={route}
+                guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_EXPENSIFY_CARD}
+                shouldShowOfflineIndicatorInWideScreen
+            >
+                <View style={[styles.mt3, isSmallScreenWidth ? styles.workspaceSectionMobile : styles.workspaceSection]}>
+                    <FeatureList
+                        menuItems={tripsFeatures}
+                        title={translate('workspace.moreFeatures.expensifyCard.feed.title')}
+                        subtitle={translate('workspace.moreFeatures.expensifyCard.feed.subTitle')}
+                        ctaText={translate('workspace.moreFeatures.expensifyCard.feed.ctaTitle')}
+                        ctaAccessibilityLabel={translate('workspace.moreFeatures.expensifyCard.feed.ctaTitle')}
+                        onCtaPress={() => {}}
+                        illustrationBackgroundColor={theme.fallbackIconColor}
```

Comment:
I think we would use the same color used in the illustration, right?

Reply:
yeah - i think it should be this one <rect width="680" height="220" fill="#085239"/>?
which equal fallbackIconColor: colors.green700 - am I right @shawnborton

Reply:
Yup, that's correct